extensions-contrib/graphite-emitter/src/main/java/io/druid/emitter/graphite/GraphiteEmitter.java

```diff
@@ -175,11 +173,12 @@ public void run()
             if (e instanceof InterruptedException) {
               Thread.currentThread().interrupt();
             } else if (e instanceof SocketException) {
+              pickledGraphite.close();
```

Comment:
General `Closeable` contract in Java is that after `close()` the object is useless: either doesn't react to any subsequent calls or throws runtime exceptions. If this is not so for `PickledGraphite`, maybe it should be commented.